db/migrate/20151021194020_add_time_of_day_to_scheduled_messages.rb

```diff
@@ -0,0 +1,5 @@
+class AddTimeOfDayToScheduledMessages < ActiveRecord::Migration
+  def change
+    add_column :scheduled_messages, :time_of_day, :time, null: false, default: "12:00:00"
```

Comment:
ah yes, makes total sense. Maybe in that case we should set the default time for users to EST so people don't get messages w default time at weird times? ps - UGH timezones! :clock1: